src/js/electron/tron/session.js

```diff
@@ -25,20 +25,21 @@ export default function session(path: string = sessionStateFile()) {
 
     load: async function(): Promise<?SessionState> {
       const migrator = await tron.migrations()
-      const saved = await lib
-        .file(path)
-        .read()
-        .then(JSON.parse)
-        .then((state) => migrate(state, migrator))
-        .catch((e) => {
-          log.error("Unable to load session state")
-          log.error(e)
-          return freshState(migrator.getLatestVersion())
-        })
-
-      if (saved) {
-        version = saved.version
-        return saved.data
+      const file = lib.file(path)
+
+      version = migrator.getLatestVersion()
```

Comment:
I realized that in all cases, we are going to set the version to the latest version. So we just do that right here instead of spreading it all around.

If migrations succeed, version will be the latest.
If migrations fail, we return undefined and set the version to the latest.
If the file does not exist, we return undefined and set version to the latest.

Reply:
Nice, make sense